docker/build/installers/py3_requirements.txt

```diff
@@ -1,6 +1,32 @@
+# system utils
+supervisor
+
 # Google infras.
 absl-py
+glog
+grpcio-tools == 1.2.0
```

Comment:
Could you remind me why we need grpcio-tools with a fixed version of 1.2.0... I remember I introduced this one previously due to Bazel/Protobuf/Grpc version issue but not sure if this is the only place needs a `grpcio-tools == 1.2.0`

Reply:
confirmed this one is not needed anymore, so will just remove it